nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js

```diff
@@ -580,14 +619,24 @@ export function EPCEdit(props: Props) {
   const ctx = useContext(GatewayContext);
 
   const handleEPCChange = (key: string, val) => {
-    setEPCConfig({...EPCConfig, [key]: val});
+    if (key === 'nat_enabled' && val) {
+      setEPCConfig({
+        ...EPCConfig,
+        [key]: val,
+        ['sgi_management_iface_vlan']: '',
+      });
```

Comment:
We shouldn't have to be doing this. Are you doing this due to flow errors ? 
Can you rebase to latest changes and see if the flow errors are gone